### PR TITLE
Version 2

### PR DESCRIPTION
**Pull request by ericgaspar**

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

File: manifest.toml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+ram.runtime = "50M"
+
+[install]
+    [install.domain]
+    type = "domain"
+    full_domain = true
+
+    [install.init_main_permission]
+    help.fr = "Cette page sera accessible par n'importe qui. En revanche, la configuration reste personnelle"
+    help.en = "The page will be accessible by anyone."
```
Comment (alexAubin) [inline manifest.toml]:
```suggestion
    help.fr = "Si 'visiteurs' est sélectionné, cette page sera accessible par n'importe qui. En revanche, la configuration reste personnelle"
    help.en = "If 'visitors' is selected, the page will be accessible by anyone. However, the configuration will still remain private."
```

File: scripts/install [hunk 2]
```diff
@@ -9,131 +9,31 @@
...
 ynh_script_progression --message="Configuring PHP-FPM..." --weight=3
 
 # Create a dedicated PHP-FPM config
-ynh_add_fpm_config
-
-#=================================================
-# NGINX CONFIGURATION
-#=================================================
-ynh_script_progression --message="Configuring NGINX web server..." --weight=5
+ynh_add_fpm_config --usage=low --footprint=low
```
Comment (alexAubin) [inline scripts/install]:
```suggestion
ynh_add_fpm_config
```

(assuming Yunohost 11.2)

File: scripts/upgrade [hunk 3]
```diff
@@ -67,53 +24,26 @@ then
...
 ynh_script_progression --message="Upgrading PHP-FPM configuration..." --weight=1
 
 # Create a dedicated PHP-FPM config
-ynh_add_fpm_config
-
-#=================================================
-# NGINX CONFIGURATION
-#=================================================
-ynh_script_progression --message="Upgrading NGINX web server configuration..." --weight=1
+ynh_add_fpm_config --usage=low --footprint=low
```
Comment (alexAubin) [inline scripts/upgrade]:
```suggestion
ynh_add_fpm_config
```

(Assuming Yunohost 11.2)

File: manifest.toml [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+[upstream]
+license = "Apache-2.0"
+website = "https://xulops.net/forge/cultivons.php"
+demo = "https://cultivons-demo.xulops.net"
+admindoc = "https://xulops.net/forge/cultivons.php?menu=about"
+userdoc = "https://xulops.net/forge/cultivons.php?menu=about"
+code = "https://xulops.net/forge/cultivons.php?menu=download"
+
+[integration]
+yunohost = ">= 11.1.20"
```
Comment (alexAubin) [inline manifest.toml]:
```suggestion
yunohost = ">= 11.2"
```

**Review by alexAubin — APPROVED** [review]

**Pull request merged**